bot/cogs/free.py

```diff
@@ -0,0 +1,123 @@
+import logging
+from datetime import datetime
+
+from discord import Colour, Embed, Member, utils
+from discord.ext import commands
+from discord.ext.commands import BucketType, Context, command, cooldown
+
+from bot.constants import Categories, Free, Roles
+
+
+log = logging.getLogger(__name__)
+
+TIMEOUT = Free.activity_timeout
+
+
+class Free:
+    """Tries to figure out which help channels are free."""
+
+    PYTHON_HELP_ID = Categories.python_help
+
+    @command(name="free", aliases=('f',))
+    @cooldown(1, 60.0, BucketType.channel)
+    async def free(self, ctx: Context, user: Member = None, seek: int = 2):
```

Comment:
Might need a better way to handle these inputs, d.py can [only handle one kwarg](https://discordpy.readthedocs.io/en/rewrite/ext/commands/commands.html#keyword-only-arguments) so if member is missing then this will error.

Reply:
Ah ok, gotcha